README.md

```diff
@@ -1 +1,9 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+> this is a simple project where I am testing the tools such as the linters, a descriptive "readme file" and the git commands so I can work as a professional for further projects
+
+## Built With
+
+- Html
+- Css
+- Linters
```

Comment:
In order to have a professional repo, we need to stick to the template provided by Microverse;

- [ ] Kindly, implement back all the sections of the README template and populate the sections with relevant information (as for exemple: your GitHub account, LinkedIn account, steps for an unprofessional user to clone your repo etc.);

The README template can be found here: [template](https://github.com/microverseinc/readme-template);